core/src/test/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -357,9 +385,16 @@ public static String randomRealisticUnicodeOfCodepointLength(int codePoints) {
         return array;
     }
 
+    public static String randomTimeValue(int lower, int upper) {
```

Comment:
This is what we have in master - I presume the lower limit is there so you can spit out a non 0 time value.